Gems/Atom/RHI/DX12/Code/Source/Platform/Windows/RHI/NsightAftermathGpuCrashTracker_Windows.cpp

```diff
@@ -63,15 +68,32 @@ void GpuCrashTracker::OnCrashDump(const void* pGpuCrashDump, const uint32_t gpuC
     WriteGpuCrashDumpToFile(pGpuCrashDump, gpuCrashDumpSize);
 }
 
-void GpuCrashTracker::OnShaderDebugInfo([[maybe_unused]] const void* pShaderDebugInfo, [[maybe_unused]] const uint32_t shaderDebugInfoSize)
+void GpuCrashTracker::OnShaderDebugInfo(const void* pShaderDebugInfo, const uint32_t shaderDebugInfoSize)
 {
-    AZ_UNUSED(pShaderDebugInfo);
-    AZ_UNUSED(shaderDebugInfoSize);
-
     if (!m_initialized)
     {
         return;
     }
+        
+    // Get shader debug information identifier
+    GFSDK_Aftermath_ShaderDebugInfoIdentifier identifier = {};
```

Comment:
It can generate some debug info data (which i save it to .nvdbg file) although the nsight gpu dump inspector still have problem to open shader source object. I posted a question in their forum.